mono/utils/mono-threads.c

```diff
@@ -353,147 +353,6 @@ thread_handle_destroy (gpointer data)
 	g_free (thread_handle);
 }
 
-static void*
-register_thread (MonoThreadInfo *info, gpointer baseptr)
```

Comment:
If this really needed? It will introduce a lot of history noise for no particular reason.